build/charts/theia/values.yaml

```diff
@@ -0,0 +1,79 @@
+clickhouse:
+  # -- Container image to use for the ClickHouse.
+  image: 
+    repository: "projects.registry.vmware.com/antrea/theia-clickhouse-server"
+    pullPolicy: "IfNotPresent"
+    tag: "21.11"
+  monitor:
+    # -- Determine whether to run a monitor to periodically check the ClickHouse
+    # memory usage and clean data.
+    enable: true
+    # -- The storage percentage at which the monitor starts to delete old records.
+    # Vary from 0 to 1.
+    threshold: 0.5
+    # -- The percentage of records in ClickHouse that will be deleted when the
+    # storage grows above threshold. Vary from 0 to 1.
+    deletePercentage: 0.5
+    # -- Container image to use for the ClickHouse Monitor.
+    image:
+      repository: "projects.registry.vmware.com/antrea/theia-clickhouse-monitor"
+      pullPolicy: "IfNotPresent"
+      tag: "latest"
+  secret:
+    # -- ClickHouse username. It will be stored in a secret.
+    username: "clickhouse_operator"
+    # -- ClickHouse password. It will be stored in a secret.
+    password: "clickhouse_operator_password"
+  port:
+    # -- HTTP port number for the ClickHouse service.
+    http: 8123
+    # -- TCP port number for the ClickHouse service.
+    tcp: 9000
+  # -- ClickHouse storage size. Can be a plain integer or as a fixed-point
+  # number using one of these quantity suffixes: E, P, T, G, M, K. Or the
+  # power-of-two equivalents: Ei, Pi, Ti, Gi, Mi, Ki.
+  storageSize: "8Gi"
+  # -- Time to live in seconds for data in the ClickHouse.
+  ttl: 3600
```

Comment:
as PV is introduced, I think data is expected to be kept with longer lifespan? counting ttl in seconds maybe turn out to be inconvenient. should we make it a string and support other type of `Intervals` (i.e. HOUR / DAY / WEEK etc.?)

Reply:
Thanks, it is supported now!